src/boot.tsx

```diff
@@ -22,6 +22,22 @@ export default (element: HTMLDivElement, hub: Hub) => {
     applyMiddleware(createEpicMiddleware(rootEpic))
   );
 
+  const getPackages = (packages: string) => {
+    if (packages){
+      return "\nfrom dcl_package_manager import install_packages, get_registry; install_packages(["+formatPackages(packages)+"], get_registry('/registry/registry.json'))";
+    }
+    return "";
+  };
+
+  const formatPackages = (packages: string) => {
+    let r = packages.split(",");
+    let output = "";
+    for (let i=0; i<r.length; i++){
+      output += '"'+r[i]+'", ';
+    } 
```

Comment:
I'm not sure you get what you expect here, I think you don't want to append the comma on the last one. I suggest this implementation:

```js
return _.map(r, package => `"${_.trim(package)}"`).join(', ');
```

Reply:
So this eventually ends up in a python array. And in python, it doesn't really matter if you have that last `,` or not (it's actually preferred). So I would keep it simple and not worry about it: https://github.com/datacamp/datacamp-light/pull/153#discussion_r302862576